### DISCUSSION TITLE
Git Clone Error "Fatal: unable to update url base from redirection"

### DISCUSSION THREAD

**Discussion by seffyroff** [community/community#51598] [Repositories] [answered]
### Select Topic Area

Question

### Body

On my Windows workstation I get an error when I try to clone a specific repo with a public url.  This clone works on colleagues machines, and works on my Linux workstation.  Here's the details:

```
 git clone -v https://github.com/EsotericSoftware/spine-runtimes.git?path=spine-csharp/src#4.1
Cloning into 'src#4.1'...
fatal: unable to update url base from redirection:
  asked for: https://github.com/EsotericSoftware/spine-runtimes.git?path=spine-csharp/src#4.1/info/refs&service=git-upload-pack
   redirect: https://github.com/EsotericSoftware/spine-runtimes?path=spine-csharp/src
```
I have spoken to the repo owners, and they also have no problem cloning from this url.  I guess I have some sort of misconfiguration in git?  I have tried running ```git-credentials-manager configure``` and that didn't help.

**Comment by airtower-luna** [comment, marked as answer]
What you're trying to clone isn't a valid repository URL, it's a URL to see a specific *file* in the web interface. Remove the `?` and everything behind it and the clone should work. Or go to the main page of the repository in the web interface and click the green "code" button, then copy the HTTPS URL from the popup.

**Reply by Mancent** [reply]
[]()

**Comment by hawary740** [comment]
i cant fix this error

**Reply by Aishwarya07890** [reply]
how did u do it bro
